stacks/checkbrackets.js

```diff
@@ -0,0 +1,29 @@
+const Stack = require('./Stack');
+
+function checkBrackets(stringValue) {
```

Comment:
You should make this extensible in the sense that there should be array of both openingBrackets and closing brackets
const openingBrackets = ["(", "{", "["]
const closingBrackets = [")", "}", "]"]
as mathematical operation can have so much more.

Reply:
will make another one with 3 type brackets